public/components/trace_analytics/home.tsx

```diff
@@ -133,6 +138,10 @@ export const Home = (props: HomeProps) => {
     DataSourceSelectableConfig
   >();
 
+  const DataSourceMenuView = props.dataSourceManagement?.ui?.getDataSourceMenu<
```

Comment:
Just for my knowledge, under what condition the `dataSourceManagement` is optional here?

Reply:
When data source is disabled. 
#data_source.enabled: false

Reply:
I think the datasource management is always enabled for now. I think we should check the dataSource plugin instead.

Reply:
In other word, the registration of datasource management is not following the MDS feature flag anymore. The dataSource plugin is the one to follow if we would like to check the MDS feature flag status here.